robocopkaka/bootcamp-29-project
# #158247385 Upload images on form submit

#### What does this PR do?
only upload images when users submit
#### Description of Task to be completed?
* only upload image on form submit when adding centers
* only upload image on form submit when editing centers
* modify tests
* modify `xhr-mock.js`
* add an action creator that dispatches `FETCH_CENTER_LOADING`
* add test for the action creator
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to localhost:8080 on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* a modal with the form used for adding centers should come up
* enter the details of the center
* click on `Add Center`
* the modal should close and you should see the new center on the page
#### Any background context you want to provide?
image upload was previously done on image select
#### What are the relevant pivotal tracker stories?(if applicable)
#158247385
